znai-enterprise/src/main/java/org/testingisdocumenting/znai/enterprise/storage/DocumentationStorage.java

```diff
@@ -22,8 +22,8 @@
 
 public interface DocumentationStorage {
     boolean contains(String docId);
-    void store(String docId, String version, Path generatedDocumentation);
+    void store(String docId, String version, Path generatedDocumentation, String actor);
     void prepare(String docId, String version, DocumentationPreparationProgress progress);
     long lastUpdateTime(String docId, String version);
-    void remove(String docId);
+    void remove(String docId, String actor);
```

Comment:
if you don't mind, let's use `actor` as a first parameter. so it will be like `who` and then `what`